components/OHPC_macros

```diff
@@ -34,6 +34,11 @@
 
 DocDir:    %{OHPC_PUB}/doc/contrib
 
+# Instead of having Source[1-9]: OHPC_macros in every SPEC file,
+# this sets Source999 for all SPEC files which include OHPC_macros.
+# Setting it to 999 and hoping it does not conflict with anything.
+Source999: OHPC_macros
+
```

Comment:
If we did this, I wonder if it would be worthwhile to cook up a special PR check that scans .spec files to confirm they didn't also define `Source999`?  Or should it be Source42 to ensure consistency with life, the universe, and everything.

Reply:
Interesting. 42 :smile: .  The chances that Source42 will be reached is also pretty small.

A first idea how to do that would be to use travis to do quick check on the SPEC file. We could also check for other things like %changelog. If you want I can look at a travis integration.

Reply:
At first I was thinking of standing up a separate web validator, but travis probably makes more sense. Would be awesome if you'd want to have a look at leveraging travis for some additional checks.

Reply:
I will set up travis with my clone of ohpc and will send you a link to have a look at it once it works.